src/main/java/com/powsybl/openloadflow/ac/AbstractDistributedSlackOuterLoop.java

```diff
@@ -30,11 +30,11 @@
      */
     private static final double SLACK_P_RESIDUE_EPS = Math.pow(10, -5);
 
-    protected static class ParticipatingElement<T> {
+    public static class ParticipatingElement<T> {
```

Comment:
I really want to get rid of these "public" changes, but I need to get those values somewhere else in the code
Should I just create a protected method that returns something understandable by the rest of the code ?
(returning Pair<String, Number> instead of participatingElement for example)